### PR TITLE
Implemented Core parts Endpoint Issue #22

### PR DESCRIPTION
**Pull request by efl7a**
Hey, I tried to tackle Issue #22.  Hope it is OK.

### CODE REVIEW

File: lib/spacex/capsules.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      end
+
+      def get(capsule_serial = nil)
+        return retrieve_all if capsule_serial.nil?
+
+        data = SPACEX::BaseRequest.get("capsules/#{capsule_serial}")
+        SPACEX::Capsules.new(data)
+      end
+
+      def info(capsule_serial = nil)
```
Comment (invacuo) [inline lib/spacex/capsules.rb]: Thoughts on making the other methods `private`? I took a stab at it in #36

File: lib/spacex/cores.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      end
+
+      def get(core_serial = nil)
+        return retrieve_all if core_serial.nil?
+
+        data = SPACEX::BaseRequest.get("cores/#{core_serial}")
+        SPACEX::Cores.new(data)
+      end
+
+      def info(core_serial = nil)
```
Comment (invacuo) [inline lib/spacex/cores.rb]: Thoughts on making the other methods `private`? I took a stab at it in #36

File: lib/spacex/capsules.rb (hunk 1 above)
Comment (efl7a) [reply lib/spacex/capsules.rb]: @invacuo - sounds like a good idea.  I'll refactor and submit.

**Comment by efl7a** [comment]
@rodolfobandeira - Thanks for working with me.  I haven't done a lot of collaborating, so this was a learning experience.

**Pull request merged**